packages/rancher-backup/charts/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: rancher/backup-restore-operator
-  tag: v1.0.2
+  tag: v1.0.3
```

Comment:
We will also need to increment the chart/app version in chart.yaml? So that users who already have installed this can upgrade to the new

Reply:
done

Reply:
This is the image tag, was there a change to the backup restore image ? If thats the case then the chart should probably bump to `1.0.300`. If no changes in image just bump the chart version like you have above.

Reply:
@cbron hmm I don't follow this part
> This is the image tag, was there a change to the backup restore image ?

yes this is a new image, so I can bump it straight to 1.0.300 is it?

Reply:
Yes thats fine. Wanted to make sure you weren't bumping the image just to bump the chart.